RUFAS/biophysical/manure/storage/anaerobic_lagoon.py

```diff
@@ -166,13 +166,23 @@ def _apply_methane_emissions(self, manure_temperature: float) -> tuple[float, fl
         total_methane = (
             storage_methane_from_degradable_volatile_solids + storage_methane_from_non_degradable_volatile_solids
         )
-        bedding_to_manure_non_degradable_volatile_solids_ratio = (
-            self._manure_to_process.bedding_non_degradable_volatile_solids
-            / (
-                self._manure_to_process.non_degradable_volatile_solids
-                + self._manure_to_process.bedding_non_degradable_volatile_solids
+
+        denominator = (
+            self._manure_to_process.non_degradable_volatile_solids
+            + self._manure_to_process.bedding_non_degradable_volatile_solids
+        )
+
+        if (
+            self._manure_to_process.non_degradable_volatile_solids == 0
+            or self._manure_to_process.bedding_non_degradable_volatile_solids == 0
+            or denominator == 0
+        ):
+            bedding_to_manure_non_degradable_volatile_solids_ratio = 0
```

Comment:
```suggestion
        if (
            self._manure_to_process.non_degradable_volatile_solids == 0.0
            or self._manure_to_process.bedding_non_degradable_volatile_solids == 0.0
            or denominator == 0.0
        ):
            bedding_to_manure_non_degradable_volatile_solids_ratio = 0.0
```

And doing this in the other storage child classes in the same spots will help prevent any additional MyPy errors.